policy-engine/src/graph.rs

```diff
@@ -94,3 +94,50 @@ pub(crate) fn index(req: HttpRequest<AppState>) -> Box<Future<Item = HttpRespons
         }
     }
 }
+
+#[cfg(test)]
+mod tests {
+    extern crate tokio;
+
+    use self::tokio::runtime::current_thread::Runtime;
+    use crate::graph;
+    use crate::AppState;
+    use actix_web::http;
+    use actix_web::test as atest;
```

Comment:
this doesn't seem worth the obfuscation

Reply:
Ack, it was just feeling strange to end up with `test::`. Do you prefer the full form (`actix_web::test::<foo>`) or the short one but without renaming (`test::<foo>`)?

Reply:
I'd prefer `actix_web::test` for the tests, shouldn't be too much overhead